### ISSUE TITLE
Enable to set corner radius for TextView

### ISSUE THREAD

**Issue by balazsgerlei** [CosmicMind/Material#664]
Sometimes I would like to specify a background color for the TextView and it would be nice to be able to also specify corner radius. Is it possible or if not, could you add support for it?

**Comment by danieldahan** [comment]
I would have to setup a sample project to test this, but you could try masking bounds and setting the cornerRadius property. TextView is essentially a view, which should support this. Let me know before I setup a test project :)

**Comment by balazsgerlei** [comment]
Sorry for the quite late response. I tried setting the cornerRadius property before which does not work somehow. Masking the bounds does somewhat work but it does hide the line and the floating label when set. I used the following code:

```
let path = UIBezierPath(roundedRect: textFieldSomething.bounds, byRoundingCorners: [.allCorners], cornerRadii: CGSize(width: 4.0, height: 4.0))
        let mask = CAShapeLayer()
        mask.path = path.cgPath
        textFieldSomething.layer.mask = mask
```

However since I created this issue I switched to using my own TextField implementation (mainly not because of this problem) so the solution for this is not urgent for me in any way and I do understand that this is kind of an obscure way of using this TextField implementation and does not adhere to any Material Design style. For this reason I understand if it will not be implemented and even I think that it might just add too much overhead for this project.

Still, awesome library and keep up the good work!
